test/doc/playground_test.lua

```diff
@@ -59,56 +59,70 @@ local function check_request(request, ...)
     t.assert_equals(response, {...})
 end
 
-g.test_select_customers = function()
-    check_request("crud.select('customers', {{'<=', 'age', 35}})", {
-        metadata = {
-            {name = 'id', type = 'unsigned'},
-            {name = 'bucket_id', type = 'unsigned'},
-            {name = 'name', type = 'string'},
-            {name = 'age', type = 'number'},
+local cases = {
```

Comment:
Don't know, which variant is better: define cases as functions explicitly or do it in a declarative way. Kept both variants to collect a feedback.

Reply:
I like the last one.

Reply:
Kept it, squashed.